src/Workspaces/CoreTest/WorkspaceTests/MSBuildWorkspaceTests.cs

```diff
@@ -499,7 +499,6 @@ public void TestMetadataReferenceHasBadHintPath()
             Assert.NotNull(csharpLib);
         }
 
-        [Fact(Skip = "https://roslyn.codeplex.com/workitem/452"), Trait(Traits.Feature, Traits.Features.Workspace)]
```

Comment:
I don't think you meant to remove the whole `Fact` and `Trait` attributes, probably just the Skip text from the constructor.

:-1: